go-controller/hack/lint.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
+set -x
 
```

Comment:
@dcbw do we need this `set -x`? the build is becoming noisy? thanks.

Reply:
removed